sakuma/todo/src/Todo.js

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import './Todo.css';
+
+export default function Todo() {
+    const [isShow, setIsShow] = useState(false);
+    const [list, setList] = useState(initialData);
+    const [text, setText] = useState('');
+    let nextId = 3;
```

Comment:
3は何をさしているか分かりにくいですね。

```suggestion
    let nextId = initialData.length +1 ;
```

また、`let`はグローバルなスコープがあるので、基本避けたいです。例えば、同じファイルの別なComponentで`nextId++`があれば、ここのid振りも変になります。

なので、このあたりもstateで管理した方がいいと思います。

```jsx
   const [nextId,setNextId] = useState(initialData.length+1);
```